Makefile

```diff
@@ -81,6 +81,14 @@ generate-deepcopy: $(DEEPCOPY_GEN_TARGETS) ## Generate code containing DeepCopy,
 api/%/zz_generated.deepcopy.go: bin/controller-gen $(DEEPCOPY_GEN_INPUTS)
 	controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+CONVERSION_GEN_TARGET=$(shell find api -type d -name "v*1" -exec echo {}\/zz_generated.conversion.go \;)
```

Comment:
This literally only ever returns `api/v1beta1/zz_generated.conversion.go`

If that's the intent then we can just use the single path, otherwise we need to make this find all of something?